### DISCUSSION TITLE
Dynamic code generation through DagFactory, not more than 200

### DISCUSSION THREAD

**Discussion by kuldeep27396** [apache/airflow#27109] [Q&A]
I have 20 folders and all have 100+ dags in each one of them.
One of the folders have 201 dags, but airflow dagfactory is creating an issue while parsing more than 200 dags.

What is the solution for this?

**Comment by o-nikolas** [comment]
Hey Kuldeep,

Thanks for starting a discussion!

Can you please provide more details on the error/issue you are seeing? It's difficult for folks to help you out with such little information, thanks!
